features/facebookImport/src/importer/importer-util.js

```diff
@@ -1,9 +1,12 @@
 import {
+    FileToLargerException,
     InvalidContentImportException,
     MissingContentImportException,
     MissingFileImportException,
 } from "./failed-import-exception";
 
+const FILE_SIZE_LIMIT = 5 * 1048576;
```

Comment:
Where does this number come from?

Reply:
I'd recommend to write: `5 * 1024 * 1024` for clarity. That said, the number is just an arbitrary, hard coded limit. This is just a huge hack until we can deal with larger files...

Reply:
Just went ahead and did that :D